### PR TITLE
Chore: add publish script

### PR DESCRIPTION
**Pull request by HeverFarber**

### CODE REVIEW

**Comment by HeverFarber** [comment]
[DynamoDB transport - Publish to NPM on Github release](https://trello.com/c/Wrpv4Mfx/3561-dynamodb-transport-publish-to-npm-on-github-release)

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
 node_modules/
-.idea/
\ No newline at end of file
+.idea/
+
+package-lock.json
+yarn.lock
```
Comment (RLRabinowitz) [inline .gitignore:5]: Why not upload those?

File: package.json [hunk 2]
```diff
@@ -12,6 +12,9 @@
   "bugs": {
     "url": "https://github.com/env0/winston-aws-dynamodb/issues"
   },
+  "scripts": {
+    "publish": "yarn publish"
```
Comment (RLRabinowitz) [inline package.json:16]: Any reason for `yarn` here instead of `npm`?

File: .gitignore (hunk 1 above)
Comment (HeverFarber) [reply .gitignore:5]: Because npm package not needed package-lock. package-lock is for the root project

Comment (HeverFarber) [reply .gitignore:5]: it is for CI but for package publish you want to see what happen to the users which install it

File: package.json (hunk 2 above)
Comment (HeverFarber) [reply package.json:16]: because all our commands with yarn. But I don't think is important.

**Pull request closed without merging**